.github/workflows/cwf-integ-test.yml

```diff
@@ -2,8 +2,9 @@
 
 name: CWF integ test
 
-on:  # yamllint disable-line rule:truthy
-  workflow_dispatch:
+on:
+  # yamllint disable-line rule:truthy
```

Comment:
Did you move these yamllint comments accidentally (looks like it could be an autoformat issue)? It seems the lines now trigger the linter.

Reply:
I replaced the case-by-case `truthy` configurations with one at the global level. We were either overriding it or ignoring the warning pretty much everywhere.

Reply:
I agree the check should be disabled globally, but it seems with the changes to these comments there are a lot of lint errors in this PR?

Reply:
Could you help me find the lint warnings, @sebathomas ? I checked before submitting and found just the following, which I believe were present already:

```
$ yamllint *.{yml,yaml} -c config/yamllint_config.yml 
codeql-analysis.yml
  12:1      warning  missing document start "---"  (document-start)
  65:8      warning  missing starting space in comment  (comments)

gcc-problems.yml
  10:1      warning  missing document start "---"  (document-start)

golang-build-test.yml
  158:9     warning  comment not indented like content  (comments-indentation)
```

Reply:
I'm seeing a reduction in linter errors. In the current master, `yamllint *.{yml,yaml} -c config/yamllint_config.yml ` produces 31 lines, including the "missing document start" above, and errors as well as warnings. In the new version I'm seeing 10 lines, all warnings, nothing new.

Reply:
So the reviewdog comments in the Changed Files are based on an old version of the PR and don't get removed automatically? That's what's confusing me.

Reply:
The workflow is triggered `on: pull_request_target`. That means, that the actual file which is executed on this PR itself, is the code from current `master`, where the respective change is not yet active.

To remove the warnings here, one would need a two step process.
1. Changing the "yaml lint config".
2. Changing everything else.

Reply:
I have created a single-purpose PR to change the yamllint config:

https://github.com/magma/magma/pull/13053

Reply:
> I have created a single-purpose PR to change the yamllint config: #13053

#13053 was closed. This is ok, as the linter warnings are only in this PR, and we decided to ignore them here, as the configuration will be adopted.

Reply:
I should have made clearer that mentioning the 2 step process was a "theoretical workaround", which is not necessary to apply here now, imho.
My appologies, @lucasgonze !